core/src/test/scala/integration/kafka/api/SaslSslAdminClientIntegrationTest.scala

```diff
@@ -448,8 +448,7 @@ class SaslSslAdminClientIntegrationTest extends AdminClientIntegrationTest with
     validateMetadataAndConfigs(createResult)
     val createResponseConfig = createResult.config(topic1).get().entries.asScala
 
-    val topicResource = new ConfigResource(ConfigResource.Type.TOPIC, topic1)
-    val describeResponseConfig = client.describeConfigs(List(topicResource).asJava).values.get(topicResource).get().entries.asScala
```

Comment:
I decided to fix this flaky test as well while I was at it, though it's not directly related to the changes in `describeTopics`.